app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
1. ` Ruby on Rails Tutorial Sample App`
tìm hiểu `I18n in rails` sau đó sử dụng I18n cho tất cả các text nhé
Hiện tại là toàn bộ file `*.html.erb`

2. Trong ruby sử dụng nháy đôi em nhé

3. Cách nhau 1 space thôi em